src/db/models/batch.rs

```diff
@@ -1,14 +1,20 @@
 use crate::db::RollupStatusType;
 use chrono::NaiveDateTime;
+use rust_decimal::Decimal;
 use serde::Serialize;
 
 #[derive(sqlx::FromRow, Clone, Debug, Serialize)]
-pub struct BlockBatch {
+pub struct Batch {
     pub hash: String,
     pub index: i64,
-    pub start_block_number: i64,
-    pub end_block_number: i64,
-    pub total_tx_num: i64,
+    pub start_chunk_index: i64,
+    pub end_chunk_index: i64,
+    #[sqlx(default)]
+    pub start_block_number: Option<i64>,
+    #[sqlx(default)]
+    pub end_block_number: Option<i64>,
+    #[sqlx(default)]
+    pub total_tx_num: Option<Decimal>,
```

Comment:
why `Option`?

Reply:
I separate getting `Batch` to two steps. First fetch the fields of `batch` table (and set these fields to None), then fetch these three fields from `chunk` table.